ec2/spark_ec2.py

```diff
@@ -1160,7 +1164,7 @@ def get_zones(conn, opts):
 
 # Gets the number of items in a partition
 def get_partition(total, num_partitions, current_partitions):
-    num_slaves_this_zone = total / num_partitions
+    num_slaves_this_zone = total // num_partitions
```

Comment:
Yeah, without this it passes a float to the start instances which get formatted like "1.0" and gets some xml barf back from amazon.